scopesim/effects/fits_headers.py

```diff
@@ -112,15 +113,27 @@ class ExtraFitsKeywords(Effect):
     For a list, ScopeSim will add the keywords to all extensions matching the
     specified type/name/number
 
+    The number of the extension can be used in a value by using the "艘"
```

Comment:
Could we use "@ii" or something similar instead? Can "@" be used in FITS headers?

Reply:
Sorry for putting you in this position :-). Thanks for merging it anyway now.

I'm not sure what is allowed. I think most ASCII is allowed, so I expect @ is fine.

I'd be fine with some other solution too, so feel free to change it later. At least it should be trivial to find places where 艘 is used.

I didn't want to use e.g. @ because then it would be difficult to actually use @ in a header. Or in particular, a header that uses both @ and the detector number. So then escape logic should be added or so. I thought this was easier.